compiler/rustc_symbol_mangling/src/typeid/typeid_itanium_cxx_abi.rs

```diff
@@ -957,23 +1186,30 @@ fn transform_ty<'tcx>(tcx: TyCtxt<'tcx>, ty: Ty<'tcx>, options: TransformTyOptio
         }
 
         ty::Dynamic(predicates, _region, kind) => {
-            ty = Ty::new_dynamic(
-                tcx,
-                transform_predicates(tcx, predicates, options),
-                tcx.lifetimes.re_erased,
-                *kind,
-            );
-        }
-
-        ty::Alias(..) => {
-            ty = transform_ty(
-                tcx,
-                tcx.normalize_erasing_regions(ty::ParamEnv::reveal_all(), ty),
-                options,
-            );
+            if predicates.is_fn_trait(tcx) {
```

Comment:
Please inline these functions into `typeid_itanium_cxx_abi`. I don't think those helpers need to exist, since they're never used anywhere else in the compiler.

Reply:
Done.